qms-tests/sanity/tests/documents/REQ-07.spec.ts

```diff
@@ -0,0 +1,129 @@
+import { test } from '@playwright/test'
+import {
+  attachScreenshot,
+  DocumentURI,
+  generateId,
+  getSecondPage,
+  HomepageURI,
+  PlatformSetting,
+  PlatformURI
+} from '../utils'
+import { allure } from 'allure-playwright'
+import { DocumentsPage } from '../model/documents/documents-page'
+import { DocumentDetails, DocumentRights, DocumentStatus, NewDocument } from '../model/types'
+import { DocumentContentPage } from '../model/documents/document-content-page'
+import { prepareDocumentStep } from './common-documents-steps'
+import { DocumentHistoryPage } from '../model/documents/document-history-page'
+
+test.use({
+  storageState: PlatformSetting
+})
+
+test.describe('QMS. Documents tests', () => {
+  test.beforeEach(async ({ page }) => {
+    await (await page.goto(`${PlatformURI}/${HomepageURI}`))?.finished()
+  })
+
+  test.afterEach(async ({ browser }) => {
+    const contexts = browser.contexts()
+    for (const context of contexts) {
+      await context.close()
+    }
+  })
+
+  test('TESTS-384. Create a new doc version and previous one then marked to “Obsolete”', async ({ page, browser }) => {
+    await allure.description('Requirement\nUsers need to create a new document')
```

Comment:
Minor: ...to createa a new document _version_

Reply:
I usually put there what are requirements, creating a doc for example. But sure I can put a bit more context